lib/config/app_config.dart

```diff
@@ -53,7 +53,7 @@ abstract class AppConfig {
   static bool livekitEnabledCalls = true;
   // static const livekitServiceUrl = 'https://famedly-livekit-server.teedee.dev';
   static const livekitServiceUrl = 'https://livekit-jwt.call.element.dev';
```

Comment:
Out of couriosity: are these going to be hosted services for the app? Or are users expected to run livekit alongside their own homeservers?

And maybe one side-question, if allowed: is there an interoperability with, say, call.element.io so that a group call can be shared with outside users (non-fluffy/non-matrix)?

All-in-all though, this looks pretty exciting. Thank you for working on this!

Reply:
I'm not an author, but from my humble knowledge and understanding - it shall work both ways, as livekit seems to be a protocol.

The only question / problem is that Element doesn't currently support custom livekit server (https://github.com/element-hq/element-meta/issues/2371).

I hope the server string will be configurable with FluffyChat, livekit is not that much complicated to be self-hosted.

Reply:
> if allowed: is there an interoperability with, say, call.element.io so that a group call can be shared with outside users (non-fluffy/non-matrix)?

there's some differences in the key sharing right now (to-device vs room events specifically), but once that gets solved (the fosdem demo f.ex had a patch to use room events for key sharing and be compatible with element), fluffychat should just be able to use call.element.io or any other livekit SFU for calls. (The current logic is look at older members in the call and try to use their SFU, because cross SFU communication is not a thing yet)

> I hope the server string will be configurable with FluffyChat, livekit is not that much complicated to be self-hosted.

That sounds doable. @krille-chan I assume we can add a developer setting (maybe hidden) to change the default livekit url?  Again this URL only matters if you are the one starting the call. Both element call and fluffychat should just use the SFU from any existing participants state event if there is already a call going on. 

tldr: which SFU to choose should not be a problem, key sharing implementation differences are a bigger blocker atm

Reply:
Sure we can. While I still wonder what makes more sense: Set the livekit server in the well-known for a server wide config or set it in the app as an app config?